apps/meeting/src/providers/VideoGridProvider/state.tsx

```diff
@@ -0,0 +1,319 @@
+import {
+  TargetDisplaySize,
+  VideoPreference,
+  VideoPreferences,
+  VideoSource,
+} from 'amazon-chime-sdk-js';
+import { priorityBasedPolicy } from '../../meetingConfig';
+
+interface AttendeeStatus {
+  attendeeId: string;
+  name: string;
+  videoEnabled: boolean;
+  // sharingContent: boolean;
+}
+
+export type RosterAttendeeType = {
```

Comment:
Isn't this the same as in the library? Why we redefine it https://github.com/aws/amazon-chime-sdk-component-library-react/blob/819a2e6817659f4f02a3f7b16b58d128c854fc2f/src/types/index.ts#L50?

Reply:
Cause we didn't export this from our library